### PR TITLE
Modifications to fix issues 47, 33, 30, 45

### PR DESCRIPTION
**Pull request by ghost**
1. Added x=y line when y variable is change from baseline
2. Added checkboxes to right panel to allow user to remove reference lines. Also added explanation of these lines via the right panel (as couldn't get hover text to work)
3. Removed demographic filters from right panel (similar to wat was done for central tendency)
4. Updated mapping on the READme file so that the example on the landing page will work with QT scatterplot and is updated to follow latest data. Also added meta data to this example, as it wouldn't work for me without it

### CODE REVIEW

**Review by xni7 — CHANGES_REQUESTED** [review]
Great work! One major comment and couple of minor ones

1.  Major: need to add contour lines 450, 480, 500 msec in CHG vs. BASE plot similar to https://www.ctspedia.org/wiki/pub/CTSpedia/ClinECGGraph000/1_Baseline_QTc_hr0.png
image
2. Minor:  suggest changing ID to integer
image
3. It is worth changing the mapping in [app.R](https://github.com/SafetyGraphics/qtexplorer/blob/c0b9baba8680824b648a7c42099a329a524fbd82/inst/deploy/app.R#L18) to match README.md
```yaml
ecg:
  id_col: ID
  base_col: BASE
  change_col: ''
  value_col: VALUE
  measure_col: PARAM
  measure_values:
    QT: ''
    QTcF: QTcF
    QTcB: ''
    RR: HR
    QRS: ''
  normal_col_low: ''
  normal_col_high: ''
  studyday_col: DAY
  visit_col: VISIT
  visitn_col: DAY
  tpt_col: TIME
  tptn_col: TIME2
  period_col: ''
  unit_col: UNIT
  baseline_flag_col: BASEFL
  baseline_flag_values: '1'
  treatment_col: TREAT
  analysis_flag_col: ''
  analysis_flag_values: ''
dm:
  id_col: ID
  group_col: TREAT
  sex_col: SEX
  race_col: RACE
  age_col: AGE
```

**Comment by Rosie-L** [comment]
> Great work! One major comment and couple of minor ones
> 
> 1. Major: need to add contour lines 450, 480, 500 msec in CHG vs. BASE plot similar to https://www.ctspedia.org/wiki/pub/CTSpedia/ClinECGGraph000/1_Baseline_QTc_hr0.png
>    image
> 2. Minor:  suggest changing ID to integer
>    image
> 3. It is worth changing the mapping in [app.R](https://github.com/SafetyGraphics/qtexplorer/blob/c0b9baba8680824b648a7c42099a329a524fbd82/inst/deploy/app.R#L18) to match README.md
> 
> ```yaml
> ecg:
>   id_col: ID
>   base_col: BASE
>   change_col: ''
>   value_col: VALUE
>   measure_col: PARAM
>   measure_values:
>     QT: ''
>     QTcF: QTcF
>     QTcB: ''
>     RR: HR
>     QRS: ''
>   normal_col_low: ''
>   normal_col_high: ''
>   studyday_col: DAY
>   visit_col: VISIT
>   visitn_col: DAY
>   tpt_col: TIME
>   tptn_col: TIME2
>   period_col: ''
>   unit_col: UNIT
>   baseline_flag_col: BASEFL
>   baseline_flag_values: '1'
>   treatment_col: TREAT
>   analysis_flag_col: ''
>   analysis_flag_values: ''
> dm:
>   id_col: ID
>   group_col: TREAT
>   sex_col: SEX
>   race_col: RACE
>   age_col: AGE
> ```

**Comment by ghost** [comment]
Xiao I have made your requested changes

File: inst/deploy/app.R [hunk 1]
```diff
@@ -31,13 +31,13 @@ ecg:
   normal_col_high: ''
   studyday_col: DAY
   visit_col: VISIT
-  visitn_col: DAY
+  visitn_col: VISIT
```
Comment (xni7) [inline inst/deploy/app.R:34]: i t would be helpful to add `base_col: BASE` for the Ph2b data example to run without mapping.

File: inst/deploy/app.R [hunk 2]
```diff
@@ -50,12 +50,6 @@ dm:
 "
 )
 
-eg_ph2_new <- qtexplorer::eg_ph2 %>%
```
Comment (xni7) [inline inst/deploy/app.R:53]: Not sure of the reason of removing line 53-56, it breaks the app without the ph2b data specified.

**Review by xni7 — APPROVED** [review]
Thanks for the nice work! There is one minor issue that is breaking the sample app,  as well as minor suggestion to enhance the example.

**Pull request merged**